sonar-orchestrator/src/main/java/com/sonar/orchestrator/server/ServerInstaller.java

```diff
@@ -329,6 +331,16 @@ private static void setIfNotPresent(Properties properties, String key, String va
     }
   }
 
+  private static void setAndFailIfForbiddenValuePresent(Properties properties, String key, String value, String forbiddenValue) {
+    String initialValue = properties.getProperty(key);
+    if (forbiddenValue.equals(initialValue)) {
```

Comment:
Can we rather check if initialValue "contains" forbiddenValue? I am just thinking of edge cases such as trailing forward slash etc.

Reply:
Ok, I will change to contains